gateway/handler_tar.go

```diff
@@ -25,7 +26,8 @@ func (i *handler) serveTAR(ctx context.Context, w http.ResponseWriter, r *http.R
 	// Get Unixfs file
 	file, err := i.api.GetUnixFsNode(ctx, resolvedPath)
 	if err != nil {
-		webError(w, "ipfs cat "+html.EscapeString(contentPath.String()), err, http.StatusBadRequest)
+		err = fmt.Errorf("error getting unixfs node for %s: %w", html.EscapeString(contentPath.String()), err)
+		webError(w, err, http.StatusBadRequest)
```

Comment:
unsure about this one, but following the rule of thumb, if we don't know why it is a  "bad request", then we return server error 500, no? Thoughts?
```suggestion
		webError(w, err, http.StatusInternalServerError)
```